scripts/seastar-addr2line

```diff
@@ -68,8 +69,49 @@ class Addr2Line:
         return self._read_resolved_address()
 
 class BacktraceResolver(object):
-    object_address_re = re.compile('^(?:(\w*)\W+)?(((/[^/]+)+)\+)?(0x[0-9a-f]+)\W*$', flags=re.IGNORECASE)
-    separator_re = re.compile('^\W*-+\W*$')
+
+    class BacktraceParser(object):
+        class Type(Enum):
+            ADDRESS = 1
+            SEPARATOR = 2
+
+        def __init__(self):
+            addr = "0x[0-9a-f]+"
+            path = "\S+"
+            #self.multi_address_re = re.compile(f"^(.*(?:at|backtrace):?)((?:\s+{addr})+)\s*$", flags=re.IGNORECASE)
+            token = f"(?:{path}\+)?{addr}"
+            full_addr_match = f"(?:({path})\+)?({addr})"
+            self.oneline_re = re.compile(f"^((?:.*(?:(?:at|backtrace):?|:))?(?:\s+))?({token}(?:\s+{token})*)\s*$", flags=re.IGNORECASE)
```

Comment:
Does it work with nested exceptions where each exception is a backtraced exception?

Reply:
I'm not sure.
Need to add it to the test.

Reply:
@tgrabiec I added a unit test for `throw_with_backtrace` and patched support in seastar-addr2line plus a test case for it.